### DISCUSSION TITLE
Fully open source? Self hostable?

### DISCUSSION THREAD

**Discussion by trymeouteh** [3xplcom/Core#70] [Q&A] [answered]
Is this service fully open source? Can anyone self host their own 3xpl block explorer website and API?

**Comment by Har01d** [comment, marked as answer]
We have open-sourced the core which processes blockchains, but the website and various APIs are not open-source yet.

But with the core being open-source, you can already fully scan the supported blockchains, build a simple database to feed TSV data into, and write a simple block explorer on top of it.

One of the reason the core has been made open-source is that our users can fully verify our data and the methods we're using to extract it. When you're processing a block using the core on your machine, you should be getting exactly the same data our web interface and APIs provide.
